src/components/pages/CompanyCertificates/UploadCompanyCerificate.tsx

```diff
@@ -98,166 +99,203 @@ export default function UploadCompanyCertificate({
+                  ? certificateTypes[0].certificateType
+                  : t(
+                      'content.companyCertificate.upload.certificateTypePlaceholder'
+                    )
               }
-            }}
-            label={''}
-          />
-          <Typography
-            variant="h4"
-            className="title"
-            sx={{
-              marginBottom: '20px',
-            }}
-          >
-            {t('content.companyCertificate.upload.uploadDocumentTitle')}
-          </Typography>
-          <Dropzone
-            acceptFormat={{ 'application/pdf': [] }}
-            maxFilesToUpload={1}
-            maxFileSize={2097152}
-            onChange={([file]) => {
-              setUploadedFile(file)
-            }}
-            errorText={t(
-              'content.usecaseParticipation.editUsecase.fileSizeError'
-            )}
-            DropStatusHeader={false}
-            DropArea={renderDropArea}
-          />
-          <div className="note">
-            <Typography variant="label2">
-              {t('content.companyCertificate.upload.note')}
+              onChangeItem={(e: CertificateTypes) => {
+                setSelectedCertificateType(e)
+              }}
+              keyTitle={'certificateType'}
+              disabled={certificateTypes?.length === 1}
+            />
+            <Typography variant="h4" className="title">
+              {t('content.companyCertificate.upload.certificateSiteTitle')}
             </Typography>
+            <Tooltips
+              additionalStyles={{
+                display: 'block',
+              }}
+              tooltipPlacement="bottom-end"
+              tooltipText={t('content.companyCertificate.upload.comingsoon')}
+            >
+              <span>
+                <SelectList
+                  error={false}
+                  helperText={t(
+                    'content.companyCertificate.upload.certificateSiteError'
+                  )}
+                  defaultValue={{}}
+                  items={[]}
+                  label={t(
+                    'content.companyCertificate.upload.certificateSiteLabel'
+                  )}
+                  placeholder={t(
+                    'content.companyCertificate.upload.certificateSitePlaceholder'
+                  )}
+                  onChangeItem={(e) => {
+                    // do nothing
+                  }}
+                  keyTitle={'certificateSite'}
+                  disabled={true}
+                />
+              </span>
+            </Tooltips>
+            <Typography
+              variant="h4"
+              className="title"
+              sx={{
+                marginBottom: '20px',
+              }}
+            >
+              {t('content.companyCertificate.upload.expiry')}
+            </Typography>
+            <Datepicker
+              placeholder={t(
+                'content.companyCertificate.upload.dateplaceholder'
+              )}
+              locale="en"
```

Comment:
shouldn't there go the currently selected locale?

Reply:
I tried it. But getting this error - Type 'string' is not assignable to type '"en" | "de"'